bpmsuite/bpmsuite70-executionserver-postgresql.json

```diff
@@ -260,6 +260,13 @@
             "required": false
         },
         {
+            "displayName": "Timer service data store refresh interval (in milliseconds)",
+            "description": "Sets refresh interval for the timer service data store.",
```

Comment:
IMHO it would be a good idea to add to the description the affected configuration (in this case, the `refresh-interval` attribute of `database-data-store` in `standalone-openshift.xml`)

Reply:
I can clarify this, but not sure if we should mention `standalone-openshift.xml`. It seems like an implementation detail and I don't see it mentioned in other env. vars. 

What about: Set `refresh-interval` for the EJB timer's service `database-data-store`?

Reply:
Sounds good!